sklearn/ensemble/tests/test_common.py

```diff
@@ -32,7 +32,8 @@
                     ("lr", LogisticRegression()),
                     ("svm", LinearSVC()),
                     ("rf", RandomForestClassifier()),
```

Comment:
```suggestion
                    ("rf", RandomForestClassifier(n_estimators=5, max_depth=3)),
```

Reply:
@ogrisel Thank you very much for your suggestions. Note that this PR is still WIP and I plan to finish the optimization with the appropriate measurements reported.